apps/notifications/src/modules/alert/repositories/alert/alert.repository.ts

```diff
@@ -30,6 +30,8 @@ export type AlertOutput = ChainMessageAlertOutput | DeploymentBalanceAlertOutput
 
 export type AlertType = AlertOutput["type"];
 
+const isNotSuppressed = sql`(${schema.Alert.params}->>'suppressedBySystem' IS NULL OR ${schema.Alert.params}->>'suppressedBySystem' != 'true')`;
```

Comment:
thought(non-blocking): queries with `->>` cannot use indexes but queries that use JSON match operator (`@>`) can